goss-testing/tests/ncn/goss-k8s-lb-status.yaml

```diff
@@ -0,0 +1,38 @@
+#
```

Comment:
Did you look at the `goss-k8s-all-loadbalancers-have-external-ips.yaml` test to make sure that it is not already covering what you need to cover here? What is this test covering that is being missed by that test?